packages/design/rollup.config.cjs

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable import/no-default-export */
-import multiInput from "rollup-plugin-multi-input";
```

Comment:
Updating this file to be cjs instead of js just means that rollup treats this build as a CJS build, which it still is.